test/api.test.js

```diff
@@ -0,0 +1,44 @@
+const app = require('../server/index.js');
+const supertest = require('supertest');
+const request = supertest(app);
+import 'regenerator-runtime/runtime';
+
+describe('API Endpoints', () => {
+  describe('/description/:productId Endpoint', () => {
+    test('/description/:productId endpoint will respond with 200 if productId is between 1000-1099', async (done) => {
+      const response = await request.get('/description/1001');
+      expect(response.status).toBe(200);
+      done();
+    })
+
+    test('/description/:productId endpoint will respond with 404 if productId is not found', async (done) => {
+      const response = await request.get('/description/990');
+      expect(response.status).toBe(404);
+      done();
+    })
+
+    test('/description/:productId endpoint will respond with one item description', async (done) => {
+      const res = await request.get('/description/1001');
+      expect(res.body.length).toBe(1);
+      done();
+    })
+  })
+
+  describe('/description/multiple Endpoint', () => {
+    const productIds = [1001, 1002, 1003];
+
+    test('/description/multiple endpoint will respond with 200 if array of productIds between 1000-1090 are sent', async (done) => {
+      const res = await request.post('/description/multiple').send({productIds: productIds});
```

Comment:
Same question as I have on line 34 of your `server/index.js` file.

Reply:
I'm going to go ahead and change it to a "get" request as it is not inserting any data into my database.